index.html

```diff
@@ -19,7 +19,7 @@
         shortName: "vc-data-model",
 
         // subtitle for the spec
-        subtitle: "Expressing verifiable information on the Web",
+        subtitle: "Expressing information with verifiable provenance",
```

Comment:
```suggestion
        //subtitle: "Expressing information with verifiable provenance",
```

This change suggestion removes the subtitle of the specification completely, given that we've been at this PR for about six months now without a resolution.

So, the proposal is -- remove the subtitle! Any objections?

Reply:
None from me